support-frontend/assets/pages/weekly-subscription-checkout/helpers/__tests__/deliveryDays.ts

```diff
@@ -1,53 +1,61 @@
 // ----- Imports ----- //
-import {
-	formatMachineDate,
-	formatUserDate,
-} from 'helpers/utilities/dateConversions';
-import { getWeeklyDays } from '../deliveryDays';
-// ----- Tests ----- //
-const tuesday = 1551175752198;
+import { formatMachineDate } from 'helpers/utilities/dateConversions';
+import { getWeeklyDays, productDeliveryOrStartDate } from '../deliveryDays';
 
-/* 2019-02-26 */
-const friday = 1551399618000;
+// 2019-02-26T10:09:12.198Z
+const tuesday = 1551175752198;
 
-/* 2019-03-01 */
+// 2019-03-07T00:20:18.000Z
 const thursday = 1551918018000;
 
-/* 2019-03-07 */
+// 2019-03-10T00:00:00.000Z
 const sunday = 1552176000000;
 
-/* 2019-03-10 */
+// 2019-12-09T11:38:40.247Z
 const ninthOfDecember = 1575891520247;
 
-/* 2019-12-09 */
-describe('deliveryDays', () => {
-	describe('date formatters', () => {
-		it('formatMachineDate formats date in YYYY-DD-MM format', () => {
-			expect(formatMachineDate(new Date(friday))).toEqual('2019-03-01');
-		});
-		it('formatUserDate formats date in a readable format', () => {
-			expect(formatUserDate(new Date(friday))).toEqual('Friday, March 1, 2019');
-		});
+describe('getWeeklyDays', () => {
+	it('if you order after wednesday, it delivers the Weekly on Friday fortnight', () => {
+		const days = getWeeklyDays(thursday);
+		expect(formatMachineDate(days[0]!)).toEqual('2019-03-22');
+	});
+
+	it('if you order before wednesday midnight, it delivers the Weekly on Friday week', () => {
+		const days = getWeeklyDays(tuesday);
+		expect(formatMachineDate(days[0]!)).toEqual('2019-03-08');
 	});
-	describe('getWeeklyDays', () => {
-		it('if you order after wednesday, it delivers the Weekly on Friday fortnight', () => {
-			const days = getWeeklyDays(thursday);
-			expect(formatMachineDate(days[0]!)).toEqual('2019-03-22');
-		});
-		it('if you order before wednesday midnight, it delivers the Weekly on Friday week', () => {
-			const days = getWeeklyDays(tuesday);
-			expect(formatMachineDate(days[0]!)).toEqual('2019-03-08');
-		});
-		it('if you order on a Sunday, it delivers the Weekly on Friday week', () => {
-			const days = getWeeklyDays(sunday);
-			expect(formatMachineDate(days[0]!)).toEqual('2019-03-22');
-		});
-		it('should not offer the 27th of December as a possible delivery date', () => {
-			const days = getWeeklyDays(ninthOfDecember);
-			expect(
-				days.find((d) => d.getDate() === 27 && d.getMonth() === 1),
-			).toEqual(undefined);
-			expect(days.length).toEqual(5);
-		});
+
+	it('if you order on a Sunday, it delivers the Weekly on Friday week', () => {
+		const days = getWeeklyDays(sunday);
+		expect(formatMachineDate(days[0]!)).toEqual('2019-03-22');
+	});
+
+	it('should not offer the 27th of December as a possible delivery date', () => {
+		const days = getWeeklyDays(ninthOfDecember);
+		expect(days.find((d) => d.getDate() === 27 && d.getMonth() === 1)).toEqual(
+			undefined,
+		);
+		expect(days.length).toEqual(5);
+	});
+});
+
+describe('productDeliveryOrStartDate', () => {
```

Comment:
This test produces quite a different date when using `getVoucherDays(Date.now(), paperProductOptions)[0]`  as was originally set for `HomeDelivery`. Happy that its as expected? This is what I got -> 
<img width="867" height="310" alt="image" src="https://github.com/user-attachments/assets/3dfb19ad-92e1-4c52-9e86-2838a79e6d95" />